src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/MSBuildRestoreUtility.cs

```diff
@@ -919,5 +919,80 @@ public static Task ReplayWarningsAndErrorsAsync(LockFile lockFile, ILogger logge
 
             return logger.LogMessagesAsync(logMessages);
         }
+
+        /// <summary>
+        /// Determines the restore style of a project.
+        /// </summary>
+        /// <param name="restoreProjectStyle">An optional user supplied restore style.</param>
+        /// <param name="hasPackageReferenceItems">A <see cref="bool"/> indicating whether or not the project has any PackageReference items.</param>
+        /// <param name="projectJsonPath">An optional path to the project's project.json file.</param>
+        /// <param name="projectDirectory">The full path to the project directory.</param>
+        /// <param name="projectName">The name of the project file.</param>
```

Comment:
`projectFileName`?

`projectName` would be something like `MyProject`, whereas `projectFileName` would be something like `MyProject.csproj`.

Reply:
This needs to be the name of the project without extension, so I think its good as is.